tools/plugins/com.liferay.ide.project.ui/src/com/liferay/ide/project/ui/upgrade/animated/AbstractLiferayTableViewCustomPart.java

```diff
@@ -393,6 +394,12 @@ private IPath createPreviewerFile( final String projectName, final IPath srcFile
                     tableViewElementList.add( tableViewElement );
                 }
             }
+            
+            if ( tableViewElementList.size() == 0 )
+            {
+                final String validationMessage = "No any file need to upgrade.";
+                page.triggerValidationEvent( validationMessage );
```

Comment:
The method `getInitItemsList` should only return the list.  This method should not have the side-effect of changing the UI.  Please redesign and resend, thanks!